src/screens/auth/registration/registration-form.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { AsyncError } from '../../../types';
+import useRegistrationForm from './registration-form-hook';
+import {
+  Button,
+  Container,
+  FormControl,
+  Heading,
+  Input,
+  VStack,
+} from 'native-base';
+
+interface RegistrationFormProps {
+  onSuccess: () => void;
+  onError: (error: AsyncError) => void;
+}
+
+const RegistrationForm: React.FC<RegistrationFormProps> = ({
+  onSuccess,
+  onError,
+}) => {
+  const { formik, isUpdateAccountLoading } = useRegistrationForm({
+    onRegistrationError: onError,
+    onRegistrationSuccess: onSuccess,
+  });
+
+  return (
+    <VStack space={4}>
+      <Container>
+        <Heading size="sm" fontWeight="600" color="coolGray.800">
```

Comment:
Not a big fan of overriding colors per heading instance in a page. It should be rather defined at a global level.